src/components/graph/adGroups.tsx

```diff
@@ -99,6 +98,15 @@ export const ADGroups = ({
     };
   }, [adGroups, auth?.authProvider, getGroupInfo]);
 
+  const customStyle: StylesConfig<adGroupModel> = {
+    multiValueLabel: (styles, { data }) => {
+      return {
+        ...styles,
+        color: data?.color,
```

Comment:
not sure if it really matters on this `styles` object, but objects are usually merged like this
```suggestion
        ...{ color: data?.color },
```